src/argilla/client/sdk/v1/datasets/api.py

```diff
@@ -234,6 +234,28 @@ def add_records(
     """
     url = "/api/v1/datasets/{id}/records".format(id=id)
 
+    active_user_id = None
+
+    for record in records:
+        cleaned_responses = []
+        response_without_user_id = False
+        if "responses" not in record:
+            continue
+        for response in record["responses"]:
+            if response["user_id"] is None:
+                if response_without_user_id:
+                    warnings.warn(
+                        f"Multiple responses without `user_id` found in record {record}, so just the first one will be used while the rest will be ignored."
```

Comment:
This warning will show each time we found a response without a user id and we already setup one for the active user, right?

Reply:
I think the `UserWarning` just shows once, as the default is to show just once the same warning in the same line, see https://docs.python.org/3/library/warnings.html#the-warnings-filter